### DISCUSSION TITLE
Lesson 7: Contract failed to verify

### DISCUSSION THREAD

**Discussion by Perso77** [Cyfrin/foundry-full-course-cu#86] [Q&A] [answered]
the command used: 
`forge script script/DeployFundMe.s.sol:DeployFundMe --rpc-url $(SEPOLIA_RPC_URL) --private-key $(PRIVATE_KEY) --broadcast --verify --etherscan-api-key $(ETHERSCAN_API_KEY) -vvvv`

issue: Contract failed to verify

Details: `Fail - Unable to verify. Please check if the correct constructor argument was entered.

```
constructor(address priceFeed) {
        i_owner = msg.sender;
        s_priceFeed = AggregatorV3Interface(priceFeed);
    }
```

Question: what is the command used to define the constructor parameter during deployment?

![FundMe argument](https://github.com/Cyfrin/foundry-full-course-f23/assets/116321001/6e1f55ea-e83d-4baa-9151-d7f2561ed4f2)
![contract](https://github.com/Cyfrin/foundry-full-course-f23/assets/116321001/e083af4a-4eb6-448d-a3dd-2b3d68e646ee)

**Comment by alymurtazamemon** [comment, marked as answer]
@Perso77 Please check this #57, here I have also mentioned the manual command to verify the contracts.
